src/components/UploaderConfigDialog/index.tsx

```diff
@@ -151,33 +156,50 @@ const UploaderConfigDialog: FC<Props> = ({ onClose, onDownload, open, ...rest })
         data-testid="uploader-config-dialog-header"
         variant="h1"
       >
-        Download
-        <br />
-        Configuration File
+        Download Configuration File
       </StyledHeader>
       <StyledDialogContent>
         <StyledBodyText data-testid="uploader-config-dialog-body" variant="body1">
-          Please provide the full pathway to the data files on your local system and to the file
-          manifest.
+          Please provide the full path to the data files and to the file manifest.
         </StyledBodyText>
-        <StyledForm onSubmit={handleDownload}>
+        <StyledForm onSubmit={handleSubmit(onSubmit)} id="uploader-config-dialog-form">
           <Box>
-            <StyledLabel id="data-folder-input-label">Local Path of Data Files Folder</StyledLabel>
+            <StyledLabel id="data-folder-input-label">
+              Full Path to Data Files Folder
+              <StyledAsterisk />
+              <Tooltip
+                title="Enter the full path for the Data Files folder on your local machine or S3 bucket"
+                open={undefined}
+                disableHoverListener={false}
+                data-testid="data-folder-input-tooltip"
+              />
+            </StyledLabel>
             <StyledOutlinedInput
-              {...register("dataFolder", { required: true })}
+              {...register("dataFolder", { required: "This field is required" })}
               placeholder="/Users/me/my-data-files-folder"
               data-testid="uploader-config-dialog-input-data-folder"
               inputProps={{ "aria-labelledby": "data-folder-input-label" }}
             />
+            <StyledHelperText>{errors?.dataFolder?.message}</StyledHelperText>
           </Box>
           <Box>
-            <StyledLabel id="manifest-input-label">Local Path of Manifest File</StyledLabel>
+            <StyledLabel id="manifest-input-label">
+              Full Path to Manifest File
+              <StyledAsterisk />
+              <Tooltip
+                title="Enter the full path for the File Manifest on your local machine or S3 bucket"
```

Comment:
All other tooltips in with a period, maybe we should consider adding it here as well.